### ISSUE TITLE
Binary attachment upload: Thumbnails for the attachments uploaded in SBL

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#792]

**Comment by olemartinorg** [comment]
Not quite sure what SBL is, but I'm assuming it refers to `app-frontend` (from context). Meaning, this issue is for showing thumbnails of attachments in app-frontend. While that's entirely possible, it is a lot more difficult than it might appear on the surface.

A few bunch of questions from me:
- Do we download an entire image and scale it on the frontend? Or do we ask the backend to provide us a smaller scale version of the image? Do we store/cache these scaled-down copies, or generate them on the fly each time?
- Which file types do we want to create thumbnails for? Does it make sense to thumbnail/preview images only, or do we want to thumbnail videos, documents and PDFs as well?
- For any file type, if we want/need to generate thumbnails server-side, how do we make sure the thumbnailer service is secure (i.e. if/when a vulnerability is discovered/exploited to execute arbitrary code, how do we ensure it cannot be used to elevate privileges?).

In summary, I don't think this is worth it from a security perspective. With great risks, we'd need to implement this carefully, and such a job would probably cost more than the reward is worth.
